test_substract is failing

pytest outputs:
```

========================== FAILURES ==========================
_______________________ test_subtract ________________________

    def test_subtract():
>       assert subtract(2, 3) == -1
E       assert 5 == -1
E        +  where 5 = subtract(2, 3)

example.py:16: AssertionError
================ 1 failed, 1 passed in 0.03s =================
```

--- a/.github/workflows/python-app.yml
+++ b/.github/workflows/python-app.yml
@@ -1,0 +1,36 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        pytest example.py

--- a/example.py
+++ b/example.py
@@ -8,7 +8,7 @@ def test_add():
 
 
 def subtract(a, b):
-    return a + b  # <--- fix this in step 8
+    return a - b  # <--- fix this in step 8
 
 
 # uncomment the following test in step 5

--- a/example.py
+++ b/example.py
@@ -12,5 +12,5 @@ def subtract(a, b):
 
 
 # uncomment the following test in step 5
-#def test_subtract():
-#    assert subtract(2, 3) == -1
+def test_subtract():
+    assert subtract(2, 3) == -1

